src/bootstrap/lib.rs

```diff
@@ -1331,6 +1337,11 @@ impl Build {
         self.rust_info.sha()
     }
 
+    /// Returns the commit info.
+    fn rust_commit_info(&self) -> Option<&channel::Info> {
+        self.rust_info.info()
```

Comment:
nit: I would rather use `builder.rust_info.info()` directly instead, I think it's more clear what's going on (and avoids people adding more wrapper functions in the future for submodules).